README.md

```diff
@@ -1,3 +1,40 @@
 # DGtalTools-contrib
-Various tools from the community built using DGtal
+
+Various tools from the community built using DGtal. Contrary the to DGtalTools project, these tools are not necessary final but can be considered as development tools or prototypes used to share recent research in progress.
+
+
+This project is actually organized as follows:
+
+ - [Geometry2d](#geometry2d)
+
+ - [Geometry3d](#geometry3d)
+
+ - [Visualisation](#visualisation)
+
+
+
+Geometry2d
+----------
+
+This section can contains various utilities related to 2d geometry (2d estimators, 2d contour tools, ...):
+
+   - meaningFullThickness: to display the meaningful thickness of digital contour.
+
+
+Geometry3d
+----------
+
+As the previous section but in 3d, it contains actually these tools:
+
+   - basicEditMesh: to apply basic mesh edition (scale change, mesh face contraction, face filtering).
+   - compareErrorMesh: compute hausdorf distance between two meshes.
```

Comment:
In fact it computes the hausdorff /2 from shape A to B. (not from the vertices).
By the way the computation was not full correct in all cases.I change it with a better more complete version which computes a better approximation in various cases.
And I also rename it with a better name (computeMeshDistances)